_search-plugins/caching/tiered-cache.md

```diff
@@ -0,0 +1,92 @@
+
+### Cache store name
+
+To use the OpenSearch-provided tiered spillover cache implementation, set the cache store name to `tiered_spillover` as shown in the following example:
+
+```yaml
+indices.request.cache.store.name: tiered_spillover: true
+```
+{% include copy.html %}
+
+### Setting on-heap and disk store tiers
+
+Set the on-heap and disk store tiers to `opensearch_onheap` and `ehcache_disk`, as shown in the following example:
+
+```yaml
+indices.request.cache.tiered_spillover.onheap.store.name: opensearch_onheap
+indices.request.cache.tiered_spillover.disk.store.name: ehcache_disk
+```
+The `opensearch_onheap` setting uses the built-in on-heap cache available in OpenSearch. 
+
+The `ehcache_disk` setting is the disk cache implementation from [Ehcache](https://www.ehcache.org/) which requires installing the `cache-ehcache` plugin.
+
+{% include copy.html %}
+
+### Configuring on-heap and disk stores
+
+The following table lists the cache store settings for the `opensearch_onheap` store.
+
+Setting | Data type | Default | Description
+:--- | :--- | :--- | :---
+`indices.request.cache.opensearch_onheap.size` | Percentage | 1% of the heap size | The size of the on-heap cache. Optional.
+`indices.request.cache.opensearch_onheap.expire` | Time unit | `MAX_VALUE` (disabled) | Specifies a time-to-live (TTL) for the cached results. Optional.
+
+The following table lists the disk cache store settings for the `ehcache_disk` store.
+
+Setting | Data type | Default | Description
+:--- | :--- | :--- | :---
+`indices.request.cache.ehcache_disk.max_size_in_bytes` | Long | `1073741824` (1 GB)  | Defines the size of the disk cache. Optional.
+`indices.request.cache.ehcache_disk.storage.path` | String | `""` | Defines the storage path for the disk cache. Required.
+`indices.request.cache.ehcache_disk.expire_after_access` | Time unit | `MAX_VALUE` (disabled) | Specifies a time-to-live (TTL) for the cached results. Optional.
+`indices.request.cache.ehcache_disk.alias` | String | `ehcacheDiskCache#INDICES_REQUEST_CACHE` | Specifies an alias for the disk cache. Optional.
+`indices.request.cache.ehcache_disk.segments` | Integer | `16` | Defines the number of segments the disk cache is separated into. Used for concurrency. Optional.
+`indices.request.cache.ehcache_disk.concurrency` | Integer | `1` | Defines the number of distinct write queues created for the disk store where a group of segments share a write queue. Optional.
+
+### Additional settings for the `tiered_spillover` store
+
+The following table lists additional settings for the `tiered_spillover` store setting.
+
+Setting | Data type | Default | Description
+:--- | :--- | :--- | :---
+`indices.request.cache.tiered_spillover.disk.store.policies.took_time.threshold` | Time unit | `10ms` | A policy which is used to determine whether to cache a query onto disk cache based on its took time. This is a dynamic setting. Optional.
```

Comment:
"onto disk cache" => "into a disk cache"?

Reply:
Yeah will change this.